### ISSUE TITLE
Deployments fail in GovCloud (copilot 1.27.0 and higher)

### ISSUE THREAD

**Issue by jryanmcc** [aws/copilot-cli#4778]
It looks like there may be a regression in version 1.27.0 when deploying to GovCloud west.

`copilot svc deploy --name web --env test`

During the update of the CloudFormation stack I see this error:

_Resource handler returned message: "Service was unable to process this request. (Service: AmazonECS; Status Code: 500; Error Code: ServerException; Request ID: 61897105-e675-4939-b35d-04cd80602377; Proxy: null)" (RequestToken: 210743fb-65a7-fe9e-20d4-5e94e4b5cdc4, HandlerErrorCode: GeneralServiceException)_

If it helps to debug, here is the diff... I don't know which part of this may be causing a problem with GovCloud.

```
~ Conditions:
    - IsDefaultRootPath: !Equals [!Ref RulePath, "/"]
~ Parameters:
    - Stickiness:
    -     Type: String
    -     Default: false
~ Resources:
    ~ HTTPListenerRuleWithDomain:
        ~ Metadata:
            ~ aws:copilot:description: 'An HTTP listener rule that redirects HTTP to HTTPS' -> 'An HTTP listener rule for path `/` that redirects HTTP to HTTPS'
        ~ Properties:
            ~ Conditions:
                (1 unchanged item)
                ~ - (changed item)
                  ~ PathPatternConfig:
                      ~ Values:
                          ~ - IsDefaultRootPath -> /*
                          - - - "/*"
                          - - - !Sub "/${RulePath}"
                          -   - !Sub "/${RulePath}/*"
    ~ HTTPRuleWithDomainPriorityAction:
        ~ Properties:
            - RulePath: !Ref RulePath
            + RulePath: ["/"]
    ~ HTTPSListenerRule:
        ~ Metadata:
for forwarding HTTPS traffic to your tasks' -> 'An HTTPS listener rule for path `/` that forwards HTTPS traffic to your tasks'
        ~ Properties:
            ~ Conditions:
                (1 unchanged item)
                ~ - (changed item)
                  ~ PathPatternConfig:
                      ~ Values:
                          ~ - IsDefaultRootPath -> /*
                          - - - "/*"
                          - - - !Sub "/${RulePath}"
                          -   - !Sub "/${RulePath}/*"
    ~ HTTPSRulePriorityAction:
        ~ Properties:
            - RulePath: !Ref RulePath
            + RulePath: ["/"]
    ~ RulePriorityFunction:
        ~ Properties:
            ~ Code:
                ~ S3Key: manual/scripts/custom-resources/rulepriorityfunction/584fba94a6e8a3ba8893428c31648e101fb8bf840647ce0b4505ffc86088ab50.zip -> manual/scripts/custom-resources/rulepriorityfunction/ac6830d3d4de8167bed1ce48eaf073ccbffe41076a1f88ea5c09b7b0ad71cb14.zip
    ~ Service:
        ~
  Continue with the deployment? (y/N)                                                                                                                                                                                            Properties:
            ~ DeploymentConfiguration:
                + Alarms:
                +     AlarmNames: []
                +     Enable: false
                +     Rollback: true
            ~ LoadBalancers:
                ~ - (changed item)
                  ~ ContainerName: !Ref TargetContainer -> web
                  ~ ContainerPort: !Ref TargetPort -> 443
    ~ TargetGroup:
        ~ Metadata:
            ~ aws:copilot:description: "A target group to connect the load balancer to your service" -> "A target group to connect the load balancer to your service on port 443"
        ~ Properties:
            ~ Port: !Ref TargetPort -> 443
            ~ TargetGroupAttributes:
                (1 unchanged item)
                ~ - (changed item)
                  ~ Value: !Ref Stickiness -> false
```

**Comment by bvtujo** [comment]
I see; it looks like Alarm based rollbacks may not be available in Govcloud yet. This is indeed a bug and we should fix it.

Let me see if I can get you a workaround for the time being.

**Comment by bvtujo** [comment]
You can remove the bad key by running `copilot svc override --tool yamlpatch -`. 

Then, in `copilot/web/overrides/cfn.patches.yml`, add the following code: 

```yaml
- op: remove
  path: /Resources/Service/Properties/DeploymentConfiguration/Alarms
```

A quick `copilot deploy` and you're done. 

If you supply the `diff` flag, you should see the following: 
```
~ Resources:
    ~ Service:
        ~ Properties:
            ~ DeploymentConfiguration:
                - Alarms:
                -     AlarmNames: []
                -     Enable: false
                -     Rollback: true
```

**Comment by KollaAdithya** [comment]
Fix is now released in v1.28 https://github.com/aws/copilot-cli/releases/tag/v1.28.0. Thanks @jryanmcc for reporting this!
